lms/templates/sysadmin_dashboard.html

```diff
@@ -59,6 +62,7 @@ <h2 class="instructor-nav">
       ## Translators: refers to http://git-scm.com/docs/git-log
       <a href="${reverse('gitlogs')}">${_('Git Logs')}</a>
       <a href="${reverse('sysadmin_task_queue')}" class="${modeflag.get('task_queue')}">${_('Task Queue')}</a>
+      <a href="${reverse('sysadmin_mgmt_commands')}" class="${modeflag.get('mgmt_commands')}">${_('MGMT Commands')}</a>
```

Comment:
no, modeflag.get('mgmt_commands') returns a string 'active-section', which is used as a selector in sass/course/instructor/_instructor_2 css. In any case, I am creating a new variable as per our discussion